### PR TITLE
Adding Devfest Czech Republic 2018

### PR DESCRIPTION
**Pull request by ryzizub**

### CODE REVIEW

File: _conferences/devfest-czech-republic-2018.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: "DevFest CZ 2018"
```
Comment (JakeWharton) [inline _conferences/devfest-czech-republic-2018.md]: The name should be "GDG DevFest" to match the others. The country and year are already displayed.

File: _conferences/devfest-czech-republic-2018.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+website: https://2018.devfest.cz/
+location: Prague, CZ
+
+date_start: 2018-11-09
+date_end:   2018-11-10
+
+cfp_start: 2018-07-10
+cfp_end:   2018-08-26
+cfp_site: bit.ly/DevFestCZ18-CfP
```
Comment (JakeWharton) [inline _conferences/devfest-czech-republic-2018.md]: This probably needs an `http://` or `https://` to work properly on the site.

File: _conferences/devfest-czech-republic-2018.md (hunk 1 above)
Comment (ryzizub) [reply _conferences/devfest-czech-republic-2018.md]: Edited in latest update as GDG DevFest

File: _conferences/devfest-czech-republic-2018.md (hunk 2 above)
Comment (ryzizub) [reply _conferences/devfest-czech-republic-2018.md]: https:// added

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**